src/Examples/UnitModels/Reactors/plug_flow_reactor_testing.ipynb

```diff
@@ -0,0 +1,1147 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Flowsheet PFR Simulation and Optimization of Ethylene Glycol Production\n",
+    "\n",
+    "\n",
+    "## Learning outcomes\n",
+    "\n",
+    "\n",
+    "- Call and implement the IDAES PFR unit model\n",
+    "- Construct a steady-state flowsheet using the IDAES unit model library\n",
+    "- Connecting unit models in a flowsheet using Arcs\n",
+    "- Fomulate and solve an optimization problem\n",
+    "    - Defining an objective function\n",
+    "    - Setting variable bounds\n",
+    "    - Adding additional constraints \n",
+    "\n",
+    "\n",
+    "## Problem Statement\n",
+    "\n",
+    "Following the previous example implementing a CSTR unit model, we can alter the flowsheet to use a plug flow reactor (PFR). As before, this example is adapted from Fogler, H.S., Elements of Chemical Reaction Engineering 5th ed., 2016, Prentice Hall,  p. 157-160 with the following chemical reaction, property packages and flowsheet. Unlike a CSTR which assumes well-mixed liquid behavior, the concentration profiles will vary spacially in one dimension. In actuality, following start-up flow reactor exhibit dynamic behavior as they approach a steady-state equilibrium; we will assume our system has already achieved steady-state behavior. The state variables chosen for the property package are **molar flows of each component by phase in each stream, temperature of each stream and pressure of each stream**. The components considered are: **ethylene oxide, water, sulfuric acid and ethylene glycol** and the process occurs in liquid phase only. Therefore, every stream has 4 flow variables, 1 temperature and 1 pressure variable.\n",
```

Comment:
Similarly, we probably should expand CSTR the first time. It would also be nice to embed a link to the CSTR example, but I am not sure how easy that is to do with the way our docs get built (@dangunter - any ideas?).

Typo: spatially.

Reply:
Looking at the `notebook_index.ipynb` file, there is a link to the local host file - I will attempt embedding it behind the text, and do the same for Module 1 below if it is successful.

Reply:
Update: I successfully added an internal link by copying the "localhost" link generated in `notebook_index.ipynb` and editing the example text to [text to display as blue hyperlink](URL the link opens when clicked)